.ci/append-to-github-output.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+set -e
+
+NAME=$1
+VALUE=$2
+
```

Comment:
Please add `cd $(dirname $0)` so that we can remove path from file name that we `cat`

Reply:
Please confirm that this good idea.
I don't think so

Reply:
This helps us to avoid confusing paths in yaml. This script is for a very specific use case, better to make yaml as simple as possible.

Reply:
> 

This allows us to run this script from the pwd so that we do not need to worry about paths to message file

Reply:
One day such code from yml will go to shell script. So nested script, that we introduce now, should be simple to let it used by any means from any place.